app/initializers/service-inject.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  name: 'service-inject',
+
+  initialize(application) {
+    application.inject('route', 'can', 'service:can');
```

Comment:
Do we also need this pattern for `session`?

Reply:
In the routes we rarely use the session, but we can use it where we want. What do you suggest?

Reply:
Ah, there is my answer. If we barely use it we don't need it here.

We'll see if the need arises in the future when mixins are gone

Reply:
We don't need this file actually, but then we need to say `can: service()` if every file we use it (and I don't like that for the routes). So this is actually an auto inject for all routes.

Reply:
I know :)